unlock-app/src/components/interface/MetadataTable.tsx

```diff
@@ -33,34 +36,82 @@ export function camelCaseToTitle(s: string): string {
   )
 }
 
+/**
+ * Downloads a file with the key metadata as CSV
+ * Includes the colum name in the first row
+ */
+function downloadAsCSV(columns: any, metadata: any) {
```

Comment:
I think we'll want thorough tests on this to make sure we don't accidentally send corrupt data!

Reply:
done!